src/GrafanaDashboardDeploy/src/Credentials.cs

```diff
@@ -0,0 +1,16 @@
+namespace DotNet.Grafana
+{
+    public class Credentials
+    {
+        /// <summary>
+        /// Credentials!
```

Comment:
Not a super helpful comment. :-)
I think we need the legal header in all these files too.